inspirehep/modules/orcid/exceptions.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+
+class DuplicatedExternalIdentifiersError(Exception):
+    http_status_code = 409
+    orcid_error_code = 9021
```

Comment:
Apologies for being the devil's advocate, but are we sure this will be constant? I don't remember any docs in regards to this, so maybe worth to add a source or something, so we know where it comes from? Although, I don't mind that much, I just remember the errors from ORCID sometimes seemed random...

Reply:
I asked myself the same question and hesitated a bit. But I asked ORCID engineers and they confirmed that error codes should not change over time in the same API version.

Reply:
:+1: